### ISSUE TITLE
html producer and transparency

### ISSUE THREAD

**Issue by hatsch** [rectalogic/webvfx#1]
i am using mlt for our community tv playout. i try now to use your really great plugin for dynamic real time overlays on our tv signal. the idea is to have a permanent (transparent) layer on top of the video signal and trigger showing elements using websockets.
the problem i am having now is that i find no way for the webvfx producer to give it a transparent backgrond.
css3 has support for that with background: rgba(255,255,255,0) but it seems when there is nothing to render at all webkit just returns white (which makes sense for usual browser usage)

do you think there is a way for your plugin to pass regions of frames that have nothing to show with transparency? 
if this is not possible one could still set a backgound and do some keying on the output. but that has it's own drawbacks..

thanks for your answer
stefan

**Comment by rectalogic** [comment]
You could write a WebVfx filter HTML that places the video image on the page and overlays transparent CSS3 content on top of it. I did that to implement a "banner" filter - it renders the video with a translucent colored lower-third banner containing text. You can see what the result looks like here http://docs.motionbox.com/videos/filter-Banner.mp4

**Comment by hatsch** [comment]
thank you for the tip with the WebVfx filter HTML!
works as wanted!
awesome plugin :)

**Comment by jmrunge** [comment]
Hi, 
Would it be too much asking the HTML code of the filter for the filter-Banner.mp4 file? I cant figure out how to do it... (Im failrly new to MLT and webvfx).  Thanks in advance!

**Comment by rectalogic** [comment]
OK, I added a banner sample in commit f5bccfca14b25f1a480c5d65c3d507dce0c229b9

**Comment by jmrunge** [comment]
Thank you very much! Will take a look at it and test it as soon as possible! 

PS: Your plugin rocks!

**Comment by j-b-m** [comment]
Hi,

I am the maintainer of Kdenlive - an open source video editor using MLT and we would be very interested to use webvfx for various tasks in Kdenlive (titles, etc). Having a transparent producer would be a better option for our workflow. I have created a clone repository where I committed patches that implement transparency for MLT producers when the "transparent=1" option is set on the consumer. The code may need some adjustments, but I would like to know if you would be open to include such feature in webvfx.

My repository can be found here:
https://github.com/j-b-m/webvfx

Best regards
